app/domain/authentication/authn_azure/xms_mirid.rb

```diff
@@ -0,0 +1,66 @@
+module Authentication
+  module AuthnAzure
+
+    Err = Errors::Authentication::AuthnAzure
+
+    class XmsMirid
+
+      REQUIRED_KEYS = %w(subscriptions resourcegroups providers).freeze
+
+      def initialize(xms_mirid_token_field)
+        @xms_mirid_token_field = xms_mirid_token_field
+
+        @mirid_parts_hash = mirid_parts_hash
```

Comment:
was there a reason the xms prefixed was chopped off?

Reply:
Please see @jonahx 's comment: https://github.com/cyberark/conjur/pull/1446#discussion_r396012256

wdyt?